### ISSUE TITLE
Change the tile status of lambda and ACM on list of services page in AWS

### ISSUE THREAD

**Issue by bluemel-gov** [bcgov/cloud-pathfinder#976]
**Describe the issue**
The service tile in the login page is out of date and still shows the old in progress status. 

**Which Sprint Goal is this issue related to?**
> Note: 'Milestone' is a ZenHub term that we use synonymously with 'Sprint'. 

The 'Milestone' description (created in ZenHub) should clearly list Sprint Goals. This section should indicate which 'goal' this issue is related to.

**Additional context**
See Ryan for the excel file for the 2nd bullet. 

**Definition of done**
- update status of Lambda from IN-PROGRESS to COMPLETE
- Create and link to a table of info in Airtable 
- use the information provided in lambda ticket (#889) to populate lambda tile
- update status of ACM time to COMPLETED

**Comment by bluemel-gov** [comment]
Regarding ACM, presented this at sprint review. Request to include this in slide/info on ACM
'If a team wanted to use a (public or private) certificate from AWS, and if the cryptographic standard can’t be met, they would want to apply for an exemption:
https://www2.gov.bc.ca/gov/content/governments/services-for-government/policies-procedures/im-it-standards/exemptions'

**Comment by sid-gov** [comment]
See info from the ticket #903

**Comment by ActionAnalytics** [comment]
Please close the ticket once the basic tile update is done, we will need time to go over the large volume of info provided and refine it into something that fits into the tile.  That will happen in another ticket once I'm back in fall.
